docs/basic-features/script.md

```diff
@@ -49,7 +49,11 @@ export default function Home() {
 }
 ```
 
-With `next/script`, you no longer need to wrap scripts in `next/head`. Further, `next/script` should **not** be used in `pages/_document.js` as `next/script` has client-side functionality to ensure loading order. For example:
+Now, you use `next/script` in the body of your Next.js page. It has client-side functionality that decides when and how to load the remote script based on the `strategy`.
+
+> **Note:**
+>
+> - `next/script` **must not** be placed in either a `next/head` component or in `pages/_document.js`. The client-side loader cannot function in those contexts.
```

Comment:
```suggestion
> - `next/script` **should not** be placed in either a `next/head` component or in `pages/_document.js`. The client-side loader cannot function in those contexts.
```
"cannot function in those contexts" could be more clear. Hmm.

Reply:
The severity of "must" over "should" is part of what motivated this change - because next/script currently fails silently if you put it in those places, I think the docs need to be more direct that that is something not to do.

Agreed that "cannot function in those contexts" could be more clear, though. Open to suggestions - maybe "The client-side loader will not function properly in those locations."?

Reply:
> Agreed that "cannot function in those contexts" could be more clear, though. Open to suggestions - maybe "The client-side loader will not function properly in those locations."?

i would just remove that last sentence, the important bit is the one before imho

Reply:
@stefanprobst Alright makes sense - will do that.